src/templates/events.jsx

```diff
@@ -52,10 +82,11 @@ const Events = ({
         pageTitle={data.title}
         backgroundImage={backgroundImage}
       />
-      {shouldShowUpcoming && upcomingEvents.length > 0 && (
+      {shouldShowUpcoming && featuredUpcomingEvents.length > 0 && (
```

Comment:
no need comparing `featuredUpcomingEvents` with zero, `.length` is enough for truthy checks